### ISSUE TITLE
DISK_FULL explanation

### ISSUE THREAD

**Issue by skrisna** [pcloudcom/console-client#67]
I recently installed the linux client on Debian using the build-from-source instructions in the main page. I was able to upload/download files properly and the status was reflective of it. But suddenly it changed to DISK_FULL: 

Down: Everything Downloaded| Up: 704KB/sec, Remaining: 35 files, 15.4GB 6h 25m, status is UPLOADING
Down: Everything Downloaded| Up: 768KB/sec, Remaining: 35 files, 15.4GB 5h 53m, status is UPLOADING
Down: Everything Downloaded| Up: 736KB/sec, Remaining: 35 files, 15.4GB 6h 8m, status is UPLOADING
Down: Everything Downloaded| Up: 736KB/sec, Remaining: 35 files, 15.4GB 6h 8m, status is DISK_FULL
Down: Everything Downloaded| Up: 704KB/sec, Remaining: 35 files, 15.4GB 6h 25m, status is DISK_FULL
Down: Everything Downloaded| Up: 704KB/sec, Remaining: 35 files, 15.4GB 6h 25m, status is DISK_FULL
Down: Everything Downloaded| Up: 704KB/sec, Remaining: 35 files, 15.4GB 6h 25m, status is DISK_FULL

What does it mean?

**Comment by ddssff** [comment]
Is your disk full?

**Comment by skrisna** [comment]
No. It isn't. I have used up only 2% of my diskspace. But after a while, it automatically changed the status back to UPLOADING and then READY.

**Comment by CorvusCorax** [comment]
I also got DISK_FULL errors when my home partition had only a few hundred megabytes left. By default the Cache folder is in ~/.pcloud/Cache.  For any file uploaded into the cloud storage, it is first cached entirelly in the cache folder and then uploaded, delayed. That means you need at least as much space on the home partition as the largest file you are ever going to upload to pcloud.

The solution for me was to make ~/.pcloud/Cache a symlink toa directory on a different partition with lots of free space. The DISK_FULL errors then went away.

DISK_FULL appears not only when the disk is completely full, but also when there is only little space left. I assume this is to not use up all the remaining available space with the cache, which could render your system unusable. Good design IMHO.

So not a bug, but needs better documentation.
